src/sass/modules/_m-modal.scss

```diff
@@ -11,6 +11,41 @@
   top: 0;
   width: 100%;
   z-index: 3;
+  cursor: pointer;
+
+  &-split {
+    .va-modal-inner {
+      min-width: 710px;
+    }
+    &-container {
+      display: flex;
+    }
+    &-left,
+    &-right {
+      flex: 1;
+      padding: 0 20px;
+    }
+    p:first-child,
```

Comment:
I pulled this from the react modal, but agreed.  Specificity is a good idea as the modal takes on more roles...